blueman/gui/manager/ManagerDeviceList.py

```diff
@@ -156,6 +156,25 @@ def drag_motion(self, widget, drag_context, x, y, timestamp):
 
     def on_event_clicked(self, widget, event):
 
+        if event.type == Gdk.EventType._2BUTTON_PRESS and event.button == 1:
+            path = self.get_path_at_pos(int(event.x), int(event.y))
+            if path is None:
+                return
+
+            row = self.get(path[0], "device", "connected")
+
+            if not row:
+                return
+
+            device = row["device"]
+            if self.Blueman is not None:
+                if self.menu is None:
+                    self.menu = ManagerDeviceMenu(self.Blueman)
```

Comment:
We should refactor that but fine for me to get the feature merged in the first place. :slightly_smiling_face:

Reply:
The whole thing can be done better but agreed this is something for later.